.mk/test.mk

```diff
@@ -26,8 +26,8 @@ test-silent: clean init-examples ## run tests in a silent mode (errors only outp
 	go test -json -race -v ./... | gotestfmt -hide "all"
 
 .PHONY: cover
-cover: ## display test coverage
-	go test -v -race $(shell go list ./... | grep -v /vendor/) -v -coverprofile=coverage.out
+cover: init-examples ## display test coverage
```

Comment:
Is this to be kept?

Reply:
We don't have a `/vendor/` directory, so it does nothing.

Note that we don't exclude this for the `test` target.

Yeah, I slipped in some cleanup.